content/articles/statistical-hypothesis-testing-python-implementation/index.md

```diff
@@ -0,0 +1,573 @@
+ 
+3. Null hypothesis (H0) is true but we reject it.
+4. Null hypothesis (H0) is not true, but we did not reject it.
+ 
+These cases are not desirable since we have not recognized the right hypothesis.
+ 
+[Type-1 error](https://www.thoughtco.com/difference-between-type-i-and-type-ii-errors-3126414) occurs when we reject the null hypothesis even though it was true.
+
+[Type-2 error](https://www.thoughtco.com/difference-between-type-i-and-type-ii-errors-3126414) involves accepting the null hypothesis even though it was wrong.
+
+The severity of type-1 and type-2 errors depend on the task at hand. Data analysts should, therefore, determine the errors that could impact the outcome negatively.
+ 
+Basing on the above example, its important to be sure due to the huge risks involved. For instance, we can assume that the patient does not have fibroids while in real sense she is affected.
+ 
+Basing on this example, it would be safe to conclude that the consequences of making a type-1 error is much more grave than making a type-2 error. 
+
+When making assumptions, it's crucial to use the `p-value` to determine whether to accept or reject the null hypothesis.
+
+### Confidence of null hypothesis
+A [p-value](https://www.investopedia.com/terms/p/p-value.asp) explains the likelihood of an assumption being true based on the null hypothesis. It is an abbreviation for `probability value`.
+
+Technically, the only way we can accept or reject our null hypothesis is after determining our [p-value](https://www.scribbr.com/statistics/p-value/). 
+ 
+**The smaller our p-value is, the more delicate it is to accept our null hypothesis.**
+ 
+To calculate our `p-value`, there should be a certain level of significance to the initial assumption.
+
+A smaller p-value means that there are numerous reasons why we should accept the alternative hypothesis.
+ 
+A p-value is usually within the range of 0 and 1. The threshold value is set at 0.05 before the experiments in order to avoid bias.
+
+The alpha level allows us to ascertain whether to accept or reject the null hypothesis. It refers to the probability of making a wrong decision when the ground truth(H0) is true. 
+
+The alpha level is determined by the following formula:
+
+```py
+alpha = 1 - confidence level
+```
+
+If the confidence level from what we've seen in our data is `95%`, then the alpha will be `0.05`:
+
+```py
+alpha = 1 - 0.95
+```
+
+It is important to set the alpha level correctly to avoid biased decisions, especially when we reject the null hypothesis. The standard or preferred level is 5%.
+ 
+### Statistical techniques for hypothesis testing
+The three methods to be discussed are:
+1. Chi-square test
+2. T-test
+3. ANOVA test
+ 
+#### Chi-square test
+Chi-square test is used to analyze two categorical variables in a data set. It focuses on the relationship between these two categorical variables. For example, comparing the after effects of malaria drug A to those of malaria drug B.
```

Comment:
I think the example maybe a bit vague for the reader to grasp why it is categorical. Do elaborate or change the example accordingly.